WORKSPACE

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+workspace(name = "io_opentelemetry_cpp")
```

Comment:
I'd like to get the repository name locked down early. What do people think of `io_opentelemetry_cpp`? Going by github URL it would be `com_github_open_telemetry_opentelemetry_cpp`

Reply:
I think com.github prefix is probably an overkill.. Just **io.opentelemetry.cpp** and **io_openelemetry_cpp** - would probably be appropriate? Or does it have to be the reverse of the project git location?

Reply:
Done. `io_opentelemetry_cpp` per #13 

AFAIK there isn't a convention for these names (which is part of the problem) :)